### PR TITLE
[SWE-1193] [BRANDBOOK] Wrote component Screen.

### PR DESCRIPTION
**Pull request by Olexandr25**
Screenshot 2022-07-19 at 13 13 20
Screenshot 2022-07-19 at 13 13 32

### CODE REVIEW

File: src/components/layouts/Screen/Screen.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { ScreenStyled } from './Screen.styled'
+
+const Screen = (props) => {
+  const { title, breadcrumbs, children, fullScreen, prefix, suffix } = props
```
Comment (5P4C3GR33D) [inline src/components/layouts/Screen/Screen.js:5]: Add propTypes.

File: src/components/layouts/Screen/Screen.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  return (
+    <ScreenStyled className="container">
+      {fullScreen ? (
+        <>
+          <div className="row">
```
Comment (5P4C3GR33D) [inline src/components/layouts/Screen/Screen.js:11]: Better this way, if your columns are all cw={12}:
```
<div className="row">
  <div className="col-12 mb-5">
    <h1>{title}</h1>
  </div>
       
  <div className="col-12">{children}</div>
</div>
```

File: src/components/layouts/Screen/Screen.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <div className="col-12 mb-5">
+              <h1>{title}</h1>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-12">{children}</div>
+          </div>
+        </>
+      ) : (
+        <>
```
Comment (5P4C3GR33D) [inline src/components/layouts/Screen/Screen.js]: Excess `Fragment`.

File: src/components/layouts/Screen/Screen.js (hunk 2 above)
Comment (Olexandr25) [reply src/components/layouts/Screen/Screen.js]: I use classical Bootstrap - GRID.
It does not have any cw={12}

**Review by 5P4C3GR33D — APPROVED** [review]

**Pull request closed without merging**